pkg/transformer/util_test.go

```diff
@@ -41,3 +44,35 @@ func TestQtumAmountToEthValue(t *testing.T) {
 		t.Errorf("in: %v, want: %s, got: %s", in, want, got)
 	}
 }
+
+func TestAddressesConvertion(t *testing.T) {
+	t.Parallel()
+
+	inputs := []struct {
+		qtumChain   string
+		ethAddress  string
+		qtumAddress string
+	}{
```

Comment:
Lets get a couple more tests in here. see these notes for some good test cases that I would want to ensure converted. https://github.com/hayeah/openzeppelin-solidity/blob/qtum/QTUM-NOTES.md

Reply:
- The issue is solved. Please, see commit hash ac69fa591332c2c93018bc9c55c126346c931ab5